roles/collect_modified_images/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# In order for this role to work, add the following task to your playbook
+# immediately before calling the deployment role.
+# - name: "Set the deployment start time"$a
+#   shell: "date -u '+%Y-%m-%dT%H:%M:%SZ'"
+#   register: start_time
+- name: "Create empty list for images"
+  set_fact:
+    pulled_images: []
+    events_file_path: "/tmp/events.txt"
+
+- name: "Generate events.json"
+  shell: "{{ oc_bin_path }} {% raw %}get events --field-selector reason==Pulling -o go-template='{{range .items}}{{.lastTimestamp}},{{.message}}{{\"\\n\"}}{{end}}' --sort-by={.lastTimestamp}{% endraw %}"
+  register: events
```

Comment:
This task (Generate events.json) needs to also set the KUBECONFIG variable in its environment to ensure that the correct cluster configuration is being used when running the command. Examples of this can be found in other roles, such as https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/master/roles/deploy_olm_operator/tasks/main.yml#L39-L40